autogpt/cli.py

```diff
@@ -164,6 +174,27 @@ def main(
         system_prompt = ai_config.construct_full_prompt()
         if cfg.debug_mode:
             logger.typewriter_log("Prompt:", Fore.GREEN, system_prompt)
+
```

Comment:
The main purpose of the addition here is to pull the creation of the workspace and file logger file from import time and remove them from the global namespace.